### PR TITLE
Creates new validator “disallowExtraFields”;

### PR DESCRIPTION
**Pull request by darlintonprauchner**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@ssense/restify-request-validator",
-  "version": "1.1.3",
+  "version": "1.1.4",
```
Comment (RemyJeancolas) [inline package.json]: @darlintonprauchner could you just revert this change please ?
It will be incremented automatically during the npm versioning process.

Comment (darlintonprauchner) [reply package.json]: Done.

**Review by RemyJeancolas — APPROVED** [review]
